airflow/ui/src/components/TriggerDag/TriggerDAGModal.tsx

```diff
@@ -0,0 +1,104 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Heading, VStack } from "@chakra-ui/react";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+
+import { Alert, Dialog } from "src/components/ui";
+
+import { TogglePause } from "../TogglePause";
+import TriggerDAGForm from "./TriggerDAGForm";
+import type { DagParams } from "./TriggerDag";
+import { TriggerDag as triggerDag } from "./TriggerDag";
+
+type TriggerDAGModalProps = {
+  dagDisplayName: string;
+  dagId: string;
+  isPaused: boolean;
+  onClose: () => void;
+  open: boolean;
+};
+
+const TriggerDAGModal: React.FC<TriggerDAGModalProps> = ({
+  dagDisplayName,
+  dagId,
+  isPaused,
+  onClose,
+  open,
+}) => {
+  const initialDagParams = useMemo(
+    () => ({
+      configJson: "{}",
+      dagId,
+      dataIntervalEnd: "",
+      dataIntervalStart: "",
+      runId: "",
+    }),
+    [dagId],
+  );
+
+  const [dagParams, setDagParams] = useState<DagParams>(initialDagParams);
+
+  const handleTrigger = useCallback(
+    (updatedDagParams: DagParams) => {
+      triggerDag(updatedDagParams);
+      onClose();
+    },
+    [onClose],
+  );
+
+  useEffect(() => {
+    if (!open) {
+      setDagParams(initialDagParams);
+    }
+  }, [open, initialDagParams]);
+
+  return (
+    <Dialog.Root onOpenChange={onClose} open={open} size="xl">
+      <Dialog.Content backdrop>
+        <Dialog.Header>
+          <VStack align="start" gap={4}>
+            <Heading size="xl">
+              Trigger DAG - {dagDisplayName}{" "}
+              <TogglePause dagId={dagParams.dagId} isPaused={isPaused} />
+            </Heading>
+            {isPaused ? (
+              <Alert status="warning" title="Paused DAG">
+                You can not trigger a paused DAG, Unpause DAG to trigger
```

Comment:
```suggestion
                Dag is paused. Triggering will create a Dag run but it won't start until the Dag is unpaused.
```

Reply:
So, should we remove the disable trigger button on paused DAG? Since the wording let user trigger it.

Reply:
Yes

Reply:
<img width="921" alt="image" src="https://github.com/user-attachments/assets/6a8c86a4-0e24-4d01-be4c-578b56761743">

Reply:
<img width="921" alt="image" src="https://github.com/user-attachments/assets/0db195a4-6849-40f3-a025-ef26e16b7885">

Reply:
Updated UI
<img width="921" alt="image" src="https://github.com/user-attachments/assets/82c766a9-db33-4f3a-83b9-c758dd099434">